routes/jwtauth/jwtVerify.js

```diff
@@ -0,0 +1,29 @@
+import { verify as _verify } from "jsonwebtoken";
+import jwtBlock from "../../models/jwtblock";
+import User from "../../models/user";
+  
+const jwtVerify = async (token, ignoreblock) => {
```

Comment:
Codacy found an issue: [Async arrow function has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/gh/implude/local-community-backend/pullRequest?prid=6225533)